### ISSUE TITLE
Support for special characters in release notes

### ISSUE THREAD

**Issue by DenHeligaAnden** [velopack/velopack#231]
I tried to write my release notes in Swedish (containing special characters å, ä, ö).
The release notes are in a .md file which i provided using --releaseNotes when packing. 

When I then read the releasenotes (NotesMarkdown or NotesHTML) from the VelopackAsset object in my C# program all å, ä, ö characters are replaced with �.

I assume this is because none-english characters are just not supported, unless their is a workaround that I am unfamiliar with.

**Comment by caesay** [comment]
This should be supported, so I'll look into it. Thanks!

**Comment by caesay** [comment]
Can you provide an example of what your `releases.{channel}.json` looks like?

**Comment by DenHeligaAnden** [comment]
[releases.win.json](https://github.com/user-attachments/files/17161519/releases.win.json)

Yeah sure, here is an example (using the latest version of velopack and vpk)

**Comment by remco1271** [comment]
What is the file encoding of your .md file?(UTF8, UTF8-BOM, ANSI)

**Comment by remco1271** [comment]
When trying to recreate it I can only get it if I save the markdown file as ANSI or Windows-1252  then I get out the same result.
Windows-1252 is not supported by default in .NET without the package "System.Text.Encoding.CodePages".
When looking at your file in a hex editor I see 3 times the same char(0xEFBF) what is a "REPLACEMENT CHARACTER" for what It couldn't read.

The code that does the conversion is here:
https://github.com/velopack/velopack/blob/53b111d0c51aed1f1c0a3c5e1858f62f44508cf2/src/vpk/Velopack.Packaging/PackageBuilder.cs#L201-L205

When reading the [File.ReadAll()](https://learn.microsoft.com/en-us/dotnet/api/system.io.file.readalltext) docs:
> This method attempts to automatically detect the encoding of a file based on the presence of byte order marks. It automatically recognizes UTF-8, little-endian UTF-16, big-endian UTF-16, little-endian UTF-32, and big-endian UTF-32 text if the file starts with the appropriate byte order marks.

Please check that you save your markdown file as:
- UTF-8
- UTF-16
- UTF-32

Here the markdown file I did try with and the output:
[releases.win.json](https://github.com/user-attachments/files/17216192/releases.win.json)
[markdown.md](https://github.com/user-attachments/files/17216196/markdown.md)

**Comment by DenHeligaAnden** [comment]
Hi!
Thank you so much for taking the time to help me out.

Is seems like the issue was with my markdown file, I am guessing it's the encoding that was wrong. Even though when I checked the file in notepad++ and VSCode they both said it was UTF-8.

So I don't really understand why and how my markdown file got screwed up, but after opening it in VSCode (I normally work in Visual Studio) and saving from there, it seems to work.
I also tried to create a new markdown in Visual Studio to see if I could recreate the issue, but nope, it worked fine with the new file.
